piccolo/utils/pydantic.py

```diff
@@ -284,40 +285,94 @@ def create_pydantic_model(
                 column._foreign_key_meta.resolved_references._meta.tablename
             )
             field = pydantic.Field(
-                extra={
-                    "foreign_key": True,
-                    "to": tablename,
-                    "target_column": column._foreign_key_meta.resolved_target_column._meta.name,  # noqa: E501
-                    **extra,
+                json_schema_extra={
+                    "extra": {
+                        "foreign_key": True,
+                        "to": tablename,
+                        "target_column": column._foreign_key_meta.resolved_target_column._meta.name,  # noqa: E501
+                        **extra,
+                    }
                 },
                 **params,
             )
             if include_readable:
                 columns[f"{column_name}_readable"] = (str, None)
         elif isinstance(column, Text):
-            field = pydantic.Field(format="text-area", extra=extra, **params)
+            field = pydantic.Field(
+                json_schema_extra={"extra": extra, "format": "text-area"},
+                **params,
+            )
         elif isinstance(column, (JSON, JSONB)):
-            field = pydantic.Field(format="json", extra=extra, **params)
+            field = pydantic.Field(
+                json_schema_extra={"extra": extra, "format": "json"}, **params
```

Comment:
@dantownsend Thanks for the review. Just one thing. I think you need to put `**params` kwarg inside `json_schema_extra` dict in **all fields** like im my PR because [extra kwarg is deprecated ](https://docs.pydantic.dev/latest/api/fields/#pydantic.fields.Field)and you get warnings when tests are executed. Everything else looks good to me.